tests/test_library_conf.py

```diff
@@ -16,7 +16,7 @@ def test_trace_header_tags_basic(self):
         """ Test that http.request.headers.user-agent is in all web spans """
 
         for _, _, span in interfaces.library.get_spans():
```

Comment:
```suggestion
        for _, span in interfaces.library.get_root_spans():
```

And you can remove the secondary condition (actully, on some tracer, we have `parent_id==0`. `get_root_spans` handles this use case

Reply:
fixed in bbbae6e